dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/deployments/MergeConfigServerXMLTest.java

```diff
@@ -63,18 +68,295 @@ public class MergeConfigServerXMLTest {
 
     @BeforeClass
     public static void startup() throws Exception {
+        server.saveServerConfiguration();
         server.startServer();
     }
 
     @AfterClass
     public static void shutdown() throws Exception {
-        server.stopServer("CWWKO1683W"); // Invalid info element
+        server.stopServer("CWWKO1683W" // Invalid info element
+        );
     }
 
     @After
     public void cleanup() throws Exception {
-        server.deleteAllDropinApplications();
-        server.removeAllInstalledAppsForValidation();
+        server.deleteAllDropinApplications(); // Will stop all dropin apps
+        server.restoreServerConfiguration(); // Will stop all apps deployed via server.xml
+        server.removeAllInstalledAppsForValidation(); // Validates that all apps stop
+
+        // Delete everything from the apps directory
+        server.deleteDirectoryFromLibertyServerRoot("apps");
+        LibertyFileManager.createRemoteFile(server.getMachine(), server.getServerRoot() + "/apps").mkdir();
+    }
+
+    @Test
+    public void testDefaultInclusion() throws Exception {
```

Comment:
A lot of tests are duplicated between here and MergeConfigTest with the only difference being weather the config is set via mpConfig or serverXML. Consider opening a tech debt item to make both extend a common abstract class?

Reply:
We could do, I think it's borderline. There's a good bit that's the same, but there's a fair bit that's different in the setup, in exactly what we check for and in which mode each test runs in.

I think on balance it's going to be easier to follow what the test is doing if everything's in the same class, rather than having to bounce between subclasses to work out what's going on.